cli/cmd/version.go

```diff
@@ -193,12 +193,40 @@ func dailyComponentUpdateAvailable(componentName string) (bool, error) {
 	}
 }
 
+func firstTimeVersionCheck(dir, file string) error {
+	var (
+		err            error
+		currentVersion *lwupdater.Version
+	)
+	defer func() {
+		cli.Event.Feature = featDailyVerCheck
+		if err != nil {
+			cli.Event.FeatureData = map[string]interface{}{"silent_error": err.Error()}
+		}
+		cli.SendHoneyvent()
+	}()
```

Comment:
Will the CLI always stay running long enough for this to send? Looks like the daily check is called in the post-execution of the root command.

Reply:
Yup. We have workers in place to wait for this. https://github.com/lacework/go-sdk/blob/main/cli/cmd/honeyvent.go#L115-L124

Reply:
Perfect!